core/src/utils/overlays.ts

```diff
@@ -43,9 +50,17 @@ export const prepareOverlay = <T extends HTMLIonOverlayElement>(el: T) => {
   }
 };
 
-export const createOverlay = <T extends HTMLIonOverlayElement>(tagName: string, opts: object | undefined): Promise<T> => {
+export const createOverlay = <T extends HTMLIonOverlayElement>(tagName: string, opts: object | undefined, customElement?: any): Promise<T> => {
   /* tslint:disable-next-line */
   if (typeof customElements !== 'undefined') {
+    if (
```

Comment:
Can we stick to a single convention of either accessing `customElements` or `window.customElements`?

Reply:
Yes, good catch. 👍